xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererRKVA.cpp

```diff
@@ -0,0 +1,214 @@
+/*
```

Comment:
when adapting to gles this file should be renamed to RenderVAAPI-GLES or something